Get issuer and vc holder used in test data generation from test  config. (#24)
w3c/vc-di-ecdsa-test-suite: README.md

## Usage

To generate test data used in the test suite, testers are required to specify
the issuer name using the environment variable `ISSUER_NAME`.

> Additionally, vc holder name may also be specified for generating disclosed
> test credentials for ecdsa sd tests using the environment variable
> `HOLDER_NAME`. If it is not specified, `Digital Bazaar` vc holder will be used.

Reviewer: ```suggestion
In addition, VC holder name for generating disclosed test credentials for
ECDSA SD tests may be specified using the environment variable `HOLDER_NAME`.
If `$HOLDER_NAME` is not specified, `Digital Bazaar` will be used.
```

Reviewer 2: We may want to start being consistent with using `ECDSA-SD` as well.

To generate test data used in the test suite, testers are required to specify
the issuer name using the environment variable `ISSUER_NAME`.

> In addition, VC holder name for generating disclosed test credentials for
> ECDSA-SD tests may be specified using the environment variable `HOLDER_NAME`.
> If `$HOLDER_NAME` is not specified, `Digital Bazaar` will be used.

Reviewer: ```suggestion
In addition, the environment variable `HOLDER_NAME` may be used to specify
the VC holder name for generating disclosed test credentials for ECDSA-SD tests.
If `$HOLDER_NAME` is not specified, `Digital Bazaar` will be used.
```
